diff --git a/index.js b/index.js
index e5eeb353f6da8968d926cb4e502207912b6c3594..eb1370dc19030404e3def46ce13938d34a9e6214 100644
--- a/index.js
+++ b/index.js
@@ -66,7 +66,7 @@ function setup(fetch) {
             return res;
           }
         } catch (err) {
-          if (err.type === 'aborted') {
+          if (err.type === 'aborted' || err.name === 'AbortError') {
             return bail(err);
           }
           const clientError = isClientError(err);